pkg/operator/ceph/cluster/controller.go

```diff
@@ -323,6 +342,10 @@ func (c *ClusterController) configureExternalCephCluster(namespace, name string,
 }
 
 func (c *ClusterController) configureLocalCephCluster(namespace, name string, cluster *cluster, clusterObj *cephv1.CephCluster) error {
+	if err := c.commonCephClusterValidation(cluster); err != nil {
```

Comment:
This should be in `initializeCluster` instead so both local and external cluster code benefit from it.